README.md

```diff
@@ -1,6 +1,10 @@
-# Diaporama React bindings
+diaporama-react
+=========
+
+Use the awesome [diaporama](https://github.com/gre/diaporama) as a react component
```

Comment:
could you replace this to something like:

Implement a React Component to use the [diaporama](https://github.com/gre/diaporama) library.